src/masternodes/mn_checks.cpp

```diff
@@ -1329,7 +1329,14 @@ class CCustomTxApplyVisitor : public CCustomTxVisitor {
                     return minted;
 
                 CalculateOwnerRewards(*mintable.val);
-                auto res = mnview.AddBalance(*mintable.val, CTokenAmount{tokenId, amount});
+                CScript mintTo{*mintable.val};
+
+                if (!obj.to.empty()) {
```

Comment:
Usage of to needs to be behind a fork, otherwise we will have a chain split between upgraded and non-upgraded nodes when it is used. Below fork we need to reject TXs that set the to field.

Reply:
Fixed: [3dd0a9a](https://github.com/DeFiCh/ain/pull/1650/commits/3dd0a9ac8e61821361475ebc4ed656e67897abf0)